apps/studio/pages/api/projects/[ref]/run-lints.ts

```diff
@@ -0,0 +1,24 @@
+import { NextApiRequest, NextApiResponse } from 'next'
```

Comment:
this file a mistake in this branch?

Reply:
When running in local environment,  the endpoint returns 404 so I added a API page route to silence the 404.